colossalai/fx/meta_profiler/meta_registry/linear.py

```diff
@@ -0,0 +1,157 @@
+from typing import Callable, Dict, List, Tuple, Union
```

Comment:
Shall we put this whole folder to auto_parallel? @FrankLeeeee 
After reading the whole part I feel this is not related to `torch.fx` anymore

Reply:
@FrankLeeeee Let's talk about this in the afternoon